rdrf/rdrf/frontend/src/pages/proms_page/components/question.tsx

```diff
@@ -39,6 +39,11 @@ class Question extends React.Component<QuestionInterface, object> {
             mainArray = mainString;
         }
         switch(transformation) {
+            case 'italic':
+                for (const substring of mainArray) {
```

Comment:
If we're pipelining and mapping looking up "fluent" interfaces and functional idioms like map, filter - maybe if we refactor later if this becomes more complex